pkgs/dart_services/pubspec.yaml

```diff
@@ -36,6 +36,5 @@ dev_dependencies:
   test: ^1.25.7
   test_descriptor: ^2.0.1
 
-dependency_overrides:
-  dartpad_shared:
-    path: ../dartpad_shared
+# This package is managed as part of the dart-pad workspace.
```

Comment:
As a nit in these files, perhaps move the `resolution` info to before the `dependencies` section?